silkrpc/ethbackend/remote_backend.cpp

```diff
@@ -393,4 +422,43 @@ std::string RemoteBackEnd::decode_status_message(const remote::EngineStatus& sta
     }
 }
 
+NodeInfoProtocols RemoteBackEnd::decode_protocols(const nlohmann::json& json) {
```

Comment:
This method should be removed as it is unnecessary (see previous comment)

Reply:
ok